website/integrations/services/hashicorp-vault/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Hashicorp Vault
+---
+
+## What is Vault
+
+From https://vaultproject.io
+
+:::note
+Secure, store and tightly control access to tokens, passwords, certificates, encryption keys for protecting secrets and other sensitive data using a UI, CLI, or HTTP API.
+:::
+
+:::note
+This is based on authentik 2022.2.1 and Vault 1.9.3. Instructions may differ between versions. This guide does not cover vault policies. See https://learn.hashicorp.com/tutorials/vault/oidc-auth?in=vault/auth-methods for a more in depth vault guide
+:::
+
+## Preparation
+
+The following placeholders will be used:
+
+- `authentik.company` is the FQDN of authentik.
+- `vault.company` is the FQDN of Vault.
+
+### Step 1
+
+In authentik, create an _OAuth2/OpenID Provider_ (under _Resources/Providers_) with these settings:
+
+:::note
+Only settings that have been modified from default have been listed.
+:::
+
+**Protocol Settings**
+
+- Name: Vault
+- Signing Key: Select any available key
+
+- Redirect URIs/Origins:
+```
+https://vault.company/ui/vault/auth/oidc/oidc/callback
+https://vault.company/oidc/callback
```

Comment:
not sure the second one is correct, I assume that's one for local CLI-based login? In that case it should be `http://localhost:8250/oidc/callback` imo

Reply:
So, for the second one, Using vault from my laptop(not on the vault server) I had to set it to vault.company, It almost might be worth setting all three. Running vault commands on the vault server will definitely be worthwhile. I'll add it.